packaging/rpm/rb_monitor.spec

```diff
@@ -0,0 +1,49 @@
+Name:    rb_monitor
+Version: %{__version}
+Release: %{__release}%{?dist}
+
+License: GNU AGPLv3
+URL: https://github.com/redBorder/rb_monitor
+Source0: %{name}-%{version}.tar.gz
+
+BuildRequires: gcc librd-devel net-snmp-devel json-c-devel librdkafka-devel libmatheval-devel libpcap-devel
```

Comment:
We don't need libpcap, I will delete it from configure.rb_monitor and it should be removed from here too